### DISCUSSION TITLE
Rebuild an entire YODA node

### DISCUSSION THREAD

**Discussion by fj-morales** [UtrechtUniversity/yoda#227] [General]
Hi, 

I've been testing different scenarios of installing/upgrading yoda (as well as you do, I guess), and I've found that when I wipe out an irods resource server and try to rebuild it completely via ansible, the process hangs when trying to reinstall the irods server on the resources server node. This seems to be related to the fact that the SSL status connection changes while the irods-icat and irods resource server are configured with the different asible tasks. 

While this specific issue can be debugged and solved in particular, I've been wondering how reliable is the ansible playbook in case of disaster recovery, when, as in the case I exposed above, the intention is to rebuild a whole node from sratch via ansible.

Please, let me know your thoughts about this possibility.

Thanks!

**Comment by stsnel** [comment]
Hi Francisco,

I think that, generally speaking, rebuilding the software and configuration of a host on a clean image should be supported by the Ansible playbook. We welcome bug reports about situations where that doesn't work as expected, so that we can see how we can improve this.

I am not aware of any other reported problems related to rebuilding servers on a clean image.
